colorbleed/plugins/maya/load/load_yeti_cache.py

```diff
@@ -0,0 +1,166 @@
+import os
+import json
+import re
+import glob
+
+from maya import cmds
+
+from avalon import api
+from avalon.maya import lib as avalon_lib, pipeline
+from colorbleed.maya import lib
+
+
+class YetiCacheLoader(api.Loader):
+
+    families = ["colorbleed.yeticache", "colorbleed.yetiRig"]
+    representations = ["fur"]
+
+    label = "Load Yeti Cache"
+    order = -9
+    icon = "code-fork"
+    color = "orange"
+
+    def load(self, context, name=None, namespace=None, data=None):
+
+        # Build namespace
+        asset = context["asset"]
+        if namespace is None:
+            namespace = self.create_namespace(asset["name"])
+
+        # Ensure Yeti is loaded
+        if not cmds.pluginInfo("pgYetiMaya", query=True, loaded=True):
+            cmds.loadPlugin("pgYetiMaya", quiet=True)
+
+        # Get JSON
+        fname, ext = os.path.splitext(self.fname)
+        settings_fname = "{}.fursettings".format(fname)
+        with open(settings_fname, "r") as fp:
+            fursettings = json.load(fp)
+
+        # Get node name from JSON
+        nodes = []
+        for node, settings in fursettings.items():
+
+            # Create new pgYetiMaya node
+            node_name = "{}:{}".format(namespace, node)
+            yeti_node = cmds.createNode("pgYetiMaya", name=node_name)
+            cmds.connectAttr("time1.outTime", "%s.currentTime" % yeti_node)
+
+            # Apply colorbleed ID to node
+            lib.set_id(node=yeti_node,
+                       unique_id=settings["cbId"],
+                       overwrite=True)
+            settings.pop("cbId", None)
```

Comment:
The fact that `cbId` uses such custom logic compared with the other settings I'm thinking should this even be just in the settings? Should this be more explicitly referred to as the node's id?

For example:

```python
{"id": "xx/yy",
 "attribute": {"renderWidth": 1.0, 
                   "renderDensity": 2.0"}
}
```

Or does that make less sense?